feature/interface/singleton/ate_tests/singleton_test/singleton_test.go

```diff
@@ -50,8 +50,9 @@ func TestMain(m *testing.M) {
 // where WW:XX:YY:ZZ are the four octets of the IPv4 in hex.  The 0x02
 // means the MAC address is locally administered.
 const (
-	plen4 = 30
-	plen6 = 126
+	plen4                      = 30
+	plen6                      = 126
+	Ipv6ControlPacketTolerance = 6
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps
maybe we can make it shorter? something like "v6PktTolerance" 
Yan, Xiao? Thoughts?

Reply:
Not checking drops for MTU+ sized packets (they are dropped) 
 -- IIUC recommendation is to not check drops for MTU+ sized packets. If so, Isnt that the test requirement to ensure MTU+ packets are not forwarded? So shouldnt the test check for drops?

Following per the test requirement. 

For IPv4 and IPv6:
With traffic flow from ATE port-1 to ATE port-2, ensure:
For MTUs of 1500, 5000, 9212:
  - Packets with size greater than the configured MTU with DF-bit set are not transmitted. <<<<<---------
  - Packets with size of configured MTU are received.
  - Packets with size less than the configured MTU are received.

Reply:
yes, i agree the test should check for drop counter rather than unicast counters. can we add that check in another pr?